ci/install.sh

```diff
@@ -6,6 +6,19 @@ set -euo pipefail
 
 ${CXX:-g++} -v
 
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+if [ -z "${GITHUB_PR_LABEL_ESPNET1:-}" ] && [ -z "${GITHUB_PR_LABEL_ESPNET2:-}" ]; then
+    # If not Label tag ESPNET 1 or ESPNET 2 but Docker, then skip all tests.
+    if [ -n "${GITHUB_PR_LABEL_DOCKER:-}" ]; then
+        log Only Docker related modifications. Skipping tests.
+        exit 0
+    fi
+fi
+
```

Comment:
Could you remove this one? I think we should manage the condition based on PR-LABEL in github action yaml.